lms/static/sass/views/_homepage.scss

```diff
@@ -81,55 +81,96 @@ $learn-more-horizontal-position: calc(50% - 100px); // calculate the left positi
       }
 
       .course-info {
-        height: $course-info-height;
+        height: ($baseline*5.5);
```

Comment:
If we are going to use this later, create this as a variable as well

Reply:
And I see you used lots of `$baseline` times something, try to make things consistent